assets/constants.ts

```diff
@@ -46,13 +46,193 @@ export = {
   // The default language code for an exploration.
   "DEFAULT_LANGUAGE_CODE": "en",
 
-  "NEW_STRUCTURE_TO_COLORS": {
+  "ALLOWED_THUMBNAIL_BG_COLORS": {
     "chapter": ["#F8BF74", "#D68F78", "#8EBBB6", "#B3D8F1"],
     "topic": ["#C6DCDA"],
     "subtopic": ["#FFFFFF"],
     "story": ["#F8BF74", "#D68F78", "#8EBBB6", "#B3D8F1"]
   },
 
+  "ALLOWED_SVG_TAGS": [
+    "a",
+    "animate",
+    "animatecolor",
+    "animatemotion",
+    "animatetransform",
+    "circle",
+    "defs",
+    "desc",
+    "ellipse",
+    "font-face",
+    "font-face-name",
+    "font-face-src",
+    "g",
+    "glyph",
+    "hkern",
+    "image",
+    "lineargradient",
+    "line",
+    "marker",
+    "metadata",
+    "missing-glyph",
+    "mpath",
+    "path",
+    "polygon",
+    "polyline",
+    "radialgradient",
+    "rect",
+    "set",
+    "stop",
+    "svg",
+    "switch",
+    "text",
+    "title",
+    "tspan",
+    "use"
+  ],
+
+  "ALLOWED_SVG_ATTRS": [
```

Comment:
Shouldn't the attributes be tag-specific? See ATTRS_WHITELIST in html_cleaner.py.

Maybe also add a brief comment explaining how the list of tags/attrs to accept was determined.